### DISCUSSION TITLE
谁来发一下资源网站啊？

### DISCUSSION THREAD

**Discussion by SaulJWu** [opennaslab/kubespider#133] [Q&A]
除了美剧天堂，还有哪些啊？
比如我想看电影多一些

**Comment by jwcesign** [comment]
cc @miRemid

**Comment by miRemid** [comment]
如果你想要通过RSS订阅来下载，可以去看看[RSSHUB](https://github.com/DIYgod/RSSHub)，那边收集了挺多内容的

**Comment by jwcesign** [comment]
Hello，@SaulJWu，感谢使用 ，除了RSS，还有所有magnet链接和torrent链接的网站都可以下，比如电影天堂/BT天堂什么的，具体就不说了
